packages/editor-ui/src/utils/pairedItemUtils.ts

```diff
@@ -165,16 +170,43 @@ function getMapping(paths: { [item: string]: string[][] }): { [item: string]: Se
 	return mapping;
 }
 
-export function getPairedItemsMapping(executionResponse: IExecutionResponse | null): {
+function getItemsCount(runData: IRunData) {
+	let itemsCount = 0;
+
+	for (const node in runData) {
+		// eslint-disable-next-line @typescript-eslint/no-for-in-array
+		for (const taskData in runData[node]) {
+			const data = runData[node][taskData].data;
```

Comment:
Oh it's an array. We can then use `for-of` instead:

```suggestion
		for (const data of runData[node]) {
```